components/datetime/src/time_zone.rs

```diff
@@ -64,35 +64,34 @@ where
 ///
```

Comment:
Nit: Update the prose above this to update references to "patterns" with the new config enum